### ISSUE TITLE
Couldn't find "libflutter.so" error

### ISSUE THREAD

**Issue by hardysonarthy** [amorenew/alps_uhf_flutter#2]
```
Caused by: java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[[zip file "/data/app/com.wiot.alps_ax6737.uhf_plugin_example-2/base.apk"],nat
iveLibraryDirectories=[/data/app/com.wiot.alps_ax6737.uhf_plugin_example-2/lib/arm, /data/app/com.wiot.alps_ax6737.uhf_plugin_example-2/base.apk!/lib/armeabi-v7a, /system/lib, /vendor/li
b, /system/vendor/lib]]] couldn't find "libflutter.so"
```

I couldn't find the file in any of the folders.

**Comment by amorenew** [comment]
@Hrobi7198228 First this library may not work on the same hardware as UHF is a custom Chinese component 
Second I exclude it on gradle by     `exclude 'lib/arm64-v8a/libflutter.so' so maybe it has a conflict with your needs
`

**Comment by hardysonarthy** [comment]
yes my device is also Chinese component (alps ax 6737_65) is this similar to your model?
![IMG20201018012641](https://user-images.githubusercontent.com/20510554/96349252-fc818f00-10e0-11eb-85be-5803c914351f.jpg)

**Comment by amorenew** [comment]
@Hrobi7198228 you need to compare .so JNI libs to know if it same SDK or not
Chinese can put different hardware in the same device model.

**Comment by mhdarshad** [comment]
i was also getting the same errir , i tried to comment 
exclude 'lib/arm64-v8a/libflutter.so' 
in gradle file it worked
